statistics/cmsketch.go

```diff
@@ -55,11 +54,15 @@ func (c *CMSketch) insert(val *types.Datum) error {
 	return nil
 }
 
-func (c *CMSketch) query(val *types.Datum) (uint32, error) {
+func (c *CMSketch) queryValue(val *types.Datum) (uint32, error) {
```

Comment:
We don't need to use pointer for `val`.

Reply:
I think it's better to use pointer to avoid copying big object.